src/client/components/app-header/app-header.vue

```diff
@@ -1,23 +1,26 @@
 <template>
-  <header role="banner" class="app-header">
+  <header role="banner" class="app-header" :class="{ 'app-header--sticky' : menu.isSticky }">
     <nuxt-link :to="localePath('index')" class="app-header__identity">
       <img class="app-header__logo" src="/images/logo.svg" alt="" />
-      <h1 class="app-header__title">Lean Web Kit</h1>
+      <h1 class="app-header__title">Lean Web Kit<br><small>a Voorhoede product</small></h1>
+      
     </nuxt-link>
     
     <a class="a11y-sr-only" :href="`#${contentId}`">{{ $t('skip_to_content') }}</a>
     
-    <nav class="app-header__menu" :class="{ 'app-header__menu--open' : menuIsOpen }">
-      <h2 class="a11y-sr-only">{{ menu.title }}</h2>
-      <ul class="app-header__menu-list">
-        <li v-for="(item, index) in menuItems" :key="index" class="app-header__menu-item">
-          <smart-link :item="item" class="app-header__menu-link" @clicked="closeMenu" />
-        </li>
-      </ul>
-      <smart-link v-if="menu.callToAction" :item="menu.callToAction" class="button app-header__button" @clicked="closeMenu" />
-      <language-selector :locales="$i18n.locales" />
-    </nav>
-    
+      <nav class="app-header__menu" :class="{ 'app-header__menu--open' : menuIsOpen }">
+        <h2 class="a11y-sr-only">{{ menu.title }}</h2>
+        <ul class="app-header__menu-list">
+          <li v-for="(item, index) in menuItems" :key="index" class="app-header__menu-item">
+            <smart-link :item="item" class="app-header__menu-link" @clicked="closeMenu" />
```

Comment:
spread over multiple lines for legibility. Shouldn't it be`@click` instead of `@clicked`?

Reply:
Try binding the `@click` here, removing the `emit` from the component. If that doesn't work, try renaming it to `click` in the component instead of `clicked`. If that doesn't work let me know.